libs/sdk-ui-kit/package.json

```diff
@@ -88,7 +89,8 @@
         "tinycolor2": "^1.4.2",
         "ts-invariant": "^0.7.3",
         "tslib": "^2.0.0",
-        "uuid": "^8.3.2"
+        "uuid": "^8.3.2",
+        "react-select": "^3.1.0"
```

Comment:
please keep list of dependencies in alphabetical order

Reply:
done